tokenomics-explainers/003-rework-foundation.md

```diff
@@ -100,16 +105,17 @@ Zooming _all_ the way out, the total revenue is `reth_commision*reth_tvl_in_vali
 ### ETH-only Validator ROIs (Saturn 1)
 Zooming in to a single operator, the easiest way to look at the impact of lower bonds is by considering ETH-only validators and looking at their ROI. As we can see - we get dramatically higher ROIs for the same commission (or, equivalently, can get the same ROI with dramatically lower commission). The takeaway is simple from a capital efficiency point of view – smaller bonds are better.
 
+TODO -- update circle and legend text to 5%. make x axis go from 3-7%. ideally samus has the final version of code; if not, searching for "arrowstyle" in discord gives code to make an earlier version of this chart.
```

Comment:
Sorry, I think you made the final tweaks here:
https://discord.com/channels/405159462932971535/1215788197842255972/1245019226385944619

Like you said, the last I had was from the "arrowstyle" message here:
https://discord.com/channels/405159462932971535/1215788197842255972/1244787553119240292

Could I let you regenerate? I could try to go back and make changes to end up in the same style, but am working on the sankey's and pie chart right now

Reply:
I don't have the code any more then. Sadge. Anyhow -- regenerating